docs/source/how-to/custom-formats.md

```diff
@@ -0,0 +1,173 @@
+# Add Support for a Custom Format
+
+For a given "structure family" (e.g. `array`, `dataframe`) the Tiled server
+can provide the data---in whole or in part---in a variety of formats. 
+See {doc}`../tutorials/export` for a list of the formats supported out of the
+box.
+
+To teach Tiled to serve another format, first decide which structure family or
+families are appropriate for this format? For example, it make sense to export
```

Comment:
maybe remove the question mark?

Reply:
Ah yes. Left over from a grammatical refactor.